.github/workflows/jira-issue-transfer.yml

```diff
@@ -31,20 +31,26 @@ jobs:
             JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         - name: Jira Create Bug
-          if: github.event.label.name == 'ticketed' && !contains(github.event.issue.labels.*.name, 'ticketed') && contains(github.event.issue.labels.*.name, 'bug') 
+          id: bug
+          if: contains(github.event.issue.labels.*.name, 'ticketed', 'bug') && !contains(github.event.issue.labels.*.name, 'ticketed') 
           uses: atlassian/gajira-create@v3
           with:
             project: BP
             issuetype: 'Bug Report'
             summary: ${{ github.event.issue.title }}
             description: "Github Issue Link: ${{ github.event.issue.html_url}} \r\n ${{ github.event.issue.body }}"
             fields: '{"labels":["GitHubReport"]}'
-        - name: Jira Create Enhancement        
-          if: github.event.label.name == 'ticketed' && !contains(github.event.issue.labels.*.name, 'ticketed') && contains(github.event.issue.labels.*.name, 'enhancement')
+          run: echo "Issue ${{ steps.bug.outputs.issue }} was created"
```

Comment:
I don't believe you can have `with` and `run` in the same step. The action should have plenty of it's own logging that you can see in the logs section, but if you wanted to have this separate logging it probably either needs to be in it's own step with the appropriate `if` statements, or you can rearrange these to have 2 separate jobs each with their own job-level `if` statement and sequence of steps.